src/plot-data.ts

```diff
@@ -2273,6 +2273,7 @@ export abstract class PlotData {
       var canvas = document.getElementById(this.canvas_id);
 
       canvas.addEventListener('mousedown', e => {
+        console.log(e.offsetX, e.offsetY)
```

Comment:
Leftover log ?

Reply:
Removed
I like to keep it to know where my mouse is